docs/user_guide/configuring.rst

```diff
@@ -289,6 +305,65 @@ use this pattern:
 For information about configuring the sidebar's contents, see :ref:`configure-sidebar`.
 
 
+Configure the navbar items
+==========================
+
+You can choose which items show up in your navbar, as well as the order
+in which they appear. 
+
+Each menu item is a *template* in Sphinx, and you may
+add any template you wish to your navbar menu via configuring ``conf.py``.
+There are two navbar sections, corresponding to the following configuration
+values:
+
+.. code-block:: python
+
+   html_theme_options = {
+   ...
+   "navbar_left": ["list", "of", "templates"]  # Populates the left navbar
+   "navbar_menu": ["list", "of", "templates"]  # Populates the rest of the navbar
+   }
+
+By default, the following configuration is used:
+
+.. code-block:: python
+
+   html_theme_options = {
+   ...
+   "navbar_left": ["navbar-logo.html"],
+   "navbar_menu": ["navbar-menu-nav.html", "navbar-menu-search.html", "navbar-menu-buttons.html"],
+   ...
+   }
+
+If you'd like to add your own custom template to this list, you
+could do this with the following steps:
+
+1. Create an HTML file in a folder called ``_templates``. For example, if
+   you wanted to display the version of your documentation using a Jinja
+   template, you could create a file: ``_templates/version.html`` and put the
+   following in it:
+
+   .. code-block:: html
+
+      <!-- This will display the version of the docs -->
+      {{ version }}
+
+1. Now add the file to your menu items. For example:
+   
+   .. code-block:: python
+
+      html_theme_options = {
+      ...
+      "navbar_left": ["version.html", "menu-logo.html"],
+      ...
+      }
+
+.. note::
+
+   Generally speaking, navbar menu items will be **right-aligned**. However, the
+   ``navbar-menu-nav.html`` template any anything that comes before it will
+   be **left-aligned**.
```

Comment:
I am wondering here: does this signal that we should rather split it in three? And also have a `navbar_right` option for the content that is right-aligned?

Reply:
Hmm, right now the menu items and icons are in the same div, using a single class/alignment. So that's indeed a bit a bigger change to be able to split those.

Reply:
We still need to ensure that the middle and right are together in the collapsible div, so that the mobile responsiveness is correct. 

I did a quick attempt, see the commit I pushed. It seems to work the same as it does now (but didn't check if the alignment is still correct if you are using multiple custom templates)